src/deploy/functions/build.ts

```diff
@@ -366,13 +378,29 @@ function discoverTrigger(endpoint: Endpoint): backend.Triggered {
       schedule: resolveString(endpoint.scheduleTrigger.schedule),
       timeZone: resolveString(endpoint.scheduleTrigger.timeZone),
     };
-    proto.renameIfPresent(
-      bkSchedule,
-      endpoint.scheduleTrigger,
-      "retryConfig",
-      "retryConfig",
-      resolveInt
+    const bkRetry: backend.ScheduleRetryConfig = {};
```

Comment:
vsfan/danielylee: Typings didn't catch this bug - why?

Reply:
resolveInt can accept a string as an argument and return an integer, because it has to be able to handle CEL expressions. 

therefore, passing it to renameIfPresent was coincidentally able to hide this bug--the retryconfig parameters being passed in as string durations instead of as numbers--from the type system.